src/main/java/Storage.java

```diff
@@ -0,0 +1,65 @@
+import java.io.*;
+import java.util.ArrayList;
+import java.util.List;
+
+public class Storage {
+    private final String filePath;
+
+    public Storage(String filePath) {
+        this.filePath = filePath;
+    }
+
+    /**
+     * Deserializes a list of tasks stored at a given path into a list of Tasks.
+     * @return a list of Tasks that were stored at the path at <code>filePath</code>.
+     * @throws IOException if a problem was encountered while trying to access the file at <code>filePath</code>
+     */
+    // TODO: 26/8/20 Add more relevant error for parsing
+    public List<Task> load() throws IOException {
+        List<Task> tasks = new ArrayList<>();
+        try (var br = new BufferedReader(new FileReader(this.filePath))) {
```

Comment:
Should complicated statements be avoided? Maybe you can try to have it done in steps?